packages/@okta/vuepress-theme-prose/global-components/StackSelector.vue

```diff
@@ -173,7 +173,7 @@
     .stack-content {
       display: inline;
 
-      & > div, & /deep/ p {
+      & > div, &::v-deep p {
```

Comment:
How come we need to change these styles in stack selector?

Reply:
This is something I thought was addressed when I replaced node-sass but that was not committed.